Dockerfile

```diff
@@ -23,15 +28,37 @@ RUN echo "%sudo   ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers && \
 WORKDIR /home/steam
 USER steam
 
-ENV am_ark_SessionName="Ark Server" \
-    am_serverMap="TheIsland" \
-    am_ark_ServerAdminPassword="k3yb04rdc4t" \
+ENV am_ark_SessionName=Ark\ Server \
+    am_serverMap=TheIsland \
+    am_ark_ServerAdminPassword=k3yb04rdc4t \
+#    am_ark_ServerPassword= \
     am_ark_MaxPlayers=70 \
     am_ark_QueryPort=27015 \
     am_ark_Port=7778 \
     am_ark_RCONPort=32330 \
-    am_arkwarnminutes=15
+    am_ark_AltSaveDirectoryName=SavedArks \
+    am_arkwarnminutes=15 \
+    am_arkAutoUpdateOnStart=false
+#   am_ark_GameModIds= \
+#   am_arkopt_clusterid=mycluster \
+#   am_arkflag_crossplay= \
+#   am_arkflag_NoTransferFromFiltering= \
+#   am_arkflag_servergamelog= \
+#   am_arkflag_ForceAllowCaveFlyers= \
```

Comment:
Could we remove the extra commented out variables here and just make sure they are documented instead?

Reply:
👍  will do.